src/circllhist.c

```diff
@@ -369,8 +374,8 @@ int hist_bucket_cmp(hist_bucket_t h1, hist_bucket_t h2) {
   // checks if h1 < h2 on the real axis.
   if(*(uint16_t *)&h1 == *(uint16_t *)&h2) return 0;
   /* place NaNs at the beginning always */
-  if(h1.val == (int8_t)0xff) return 1;
-  if(h2.val == (int8_t)0xff) return -1;
+  if(h1.val == -1) return 1;
+  if(h2.val == -1) return -1;
```

Comment:
Why this change?  The val with all it's bits set is NaN. -1 is a tad less clear.